tests/cloudpickle_test.py

```diff
@@ -69,16 +66,16 @@ def pickle_depickle(obj, protocol=cloudpickle.DEFAULT_PROTOCOL):
     return pickle.loads(cloudpickle.dumps(obj, protocol=protocol))
 
 
-class CloudPicklerTest(unittest.TestCase):
-    def setUp(self):
-        self.file_obj = StringIO()
-        self.cloudpickler = cloudpickle.CloudPickler(self.file_obj, 2)
-
-
 class CloudPickleTest(unittest.TestCase):
 
     protocol = cloudpickle.DEFAULT_PROTOCOL
 
+    def setUp(self):
+        self.tmpdir = tempfile.mkdtemp()
```

Comment:
```suggestion
        self.tmpdir = tempfile.mkdtemp(prefix="tmp_cloudpickle_test_")
```

so that it's easier to trace the origin of uncollected temp files in case of hard-crash.